README.md

```diff
@@ -1,3 +1,4 @@
 # Sinuheguerracom2
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.1.1.
+This is a new definition.git. This is Tanya's change 
```

Comment:
Let's keep both changes. I am ok with mine going second.

Reply:
Sounds good, will resolve the conflict.